src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital@0;1&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      
+      <nav class="nav">
+        <ul class="link-list">
+          <label class="nav-item">
```

Comment:
It  should be not label.
ul coild contain only `li` tag
Fix all places
```suggestion
          <li class="nav-item">
```